llvm/lib/CodeGen/MachineBlockPlacement.cpp

```diff
@@ -3291,6 +3291,11 @@ void MachineBlockPlacement::initDupThreshold() {
 }
 
 bool MachineBlockPlacement::runOnMachineFunction(MachineFunction &MF) {
+  // Disable this pass to avoid machine blocks being merged together during
+  // codegen. Otherwise IR mapping is not preserved.
+  // FIXME: this should be behind a flag.
```

Comment:
I wonder if, in this fork, we should have `YKFIXME` so that we can grep purely for own FIXMEs?